pkg/ccl/changefeedccl/kafkaauth/scram_client.go

```diff
@@ -3,7 +3,7 @@
 // Use of this software is governed by the CockroachDB Software License
 // included in the /LICENSE file.
 
-package changefeedccl
+package kafkaauth
```

Comment:
This commit should be squashed with the third commit so that there aren't any build errors.